test/multiverse/suites/rack/rack_builder_test.rb

```diff
@@ -112,6 +112,21 @@ def instance.middleware_instrumentation_enabled?; false; end
     end
   end
 
+  # Rack::Builder#run supports blocks as of v3.0.0
+  def test_run_with_tracing_with_the_original_run_method_is_given_a_block
+    skip 'Requires Rack v3+' unless defined?(Rack::RELEASE) && Rack::RELEASE >= '3.0.0'
```

Comment:
Interesting! I didn't realize they weren't on the VERSION constant train 🚂

Reply:
There is a `Rack::VERSION` constant, but it references the [spec](https://github.com/rack/rack/blob/main/SPEC.rdoc). We need `Rack::RELEASE` to check the gem version itself.